lib/rspec/support/object_formatter.rb

```diff
@@ -2,10 +2,38 @@ module RSpec
   module Support
     # Provide additional output details beyond what `inspect` provides when
     # printing Time, DateTime, or BigDecimal
```

Comment:
If you add `# @api private` here it will apply to all the methods in the class and you won't need the individual ones. This will neatly side step the rubocop issue.